Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.16-alpine
+
+WORKDIR /app
+
+COPY go.mod ./
+COPY go.sum ./
+RUN go mod download
+
+COPY lz77/*.go ./lz77/
+COPY *.go ./
+
+RUN go build -o /xbscli-docker
```

Comment:
Why would it require a dedicated output name over using the default (`xbscli`)?

Reply:
fixed